Makefile

```diff
@@ -133,6 +133,7 @@ assets-clean:
 # Run all unit tests
 test: spec assets
 	HOME=$(CURDIR) $(PYTHON) -m pytest --continue-on-collection-errors $(TESTDIR)
+	HOME=$(CURDIR)/ocrd_utils/ocrd_utils $(PYTHON) -m pytest --continue-on-collection-errors $(TESTDIR) -k TestLogging
```

Comment:
```suggestion
	HOME=$(CURDIR)/ocrd_utils $(PYTHON) -m pytest --continue-on-collection-errors $(TESTDIR) -k TestLogging
```

Reply:
Sorry – my ultimate request, I promise!

Reply:
Promise broken already :roll_eyes: 

There's another problem which must be dealt with before this can be merged: When you set up multiple loggers via `logging.config.fileConfig` (not via `logging.getLogger`), then all [their propagate attributes](https://docs.python.org/3/library/logging.html#logging.Logger.propagate) default to true. But this means that any message they receive will _also_ go to their parents (where they usually will be rendered as a duplicate).

For example, if I set up a logger qualifying for `ocrd.processor` at debug level and a root logger at info level, then all processor messages will be rendered twice.

The remedy is simply to do `logger.propagate = False` in `ocrd_utils.logging.getLogger`. @M3ssman may I add this commit, or would you like to do it yourself?

Reply:
@bertsky Added your suggesstion ;)